content/content-strategy/cms/cms-requirements-checklist/checklist3.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: text/textblock
+---
+
+- **Hosting** 
+
+    How and where is your website hosted?
+- **Security** 
+
+    What types of security are offered?  
+    
+    Are there any possible causes for concern if your content is compromised, damaged or lost?
+    
+    What site protection features are in place?
+    
+    Does your content need to be encrypted?
+    
+    Are you using an SSL certificate for public facing web content? 
+- **Backups** 
+
+    How often are content backups done and where are they stored?
+- **Disaster Recovery** 
+
+    Is there a disaster recovery plan?
+    How are issues logged by you?
+
+- **Exit Strategy** 
+
+    If you change your CMS, is there a clear, secure process in place for [migrating content](/content-strategy/cms/optimise-cms/#exit-strategy-and-migrating-content), along with [clear CMS requirements?](/content-strategy/cms/cms-requirements/)
```

Comment:
typo for link display text, `clear CMS requirements`

Reply:
Steve. Fixed